### PR TITLE
Converted tooltip value into percentage

### PR DESCRIPTION
**Pull request by sandygross**

### CODE REVIEW

File: src/callbacks.py
Comment (zywkloo) [inline src/callbacks.py]: Looks good to me.
But since here seems to be the only place that was changed.
I would suggest only changing this part, not formatting the other part of the code.

One way to make it is to only edit this part in a pure text editor other than IDE with linter (like VScode); Or edit it in github online interface.

**Review by zywkloo — CHANGES_REQUESTED** [review]
LGTM.
```
   alt.Tooltip(
                    field=y_column,
                    type="quantitative",
                    title=f"{'Relative' if selected_type == 'relative' else 'Absolute'} {'Incidence' if selected_value == 'incidence' else 'Mortality'}",
                    format=".1%" if selected_type == "relative" else "",
                ),
```
 But since here seems to be the only place that was changed.
I would suggest only changing this part, not formatting the other part of the code.

One way to make it is to only edit this part in a pure text editor other than IDE with linter (like VScode); Or edit it in github online interface.

**Review by sean-m-mckay — APPROVED** [review]
LGTM after resolving merge conflicts

**Review by zywkloo — APPROVED** [review]

**Pull request merged**
